pyvista/plotting/renderer.py

```diff
@@ -1070,6 +1071,11 @@ def show_bounds(
             Bounds to override mesh bounds in the form ``[xmin, xmax,
             ymin, ymax, zmin, zmax]``.
 
+        axes_range : list, tuple, or np.ndarray, optional
```

Comment:
+1 for the new name.

Reply:
I'm also thinking that it should be `axes_ranges`.

Reply:
Thoughts on my updated implementation of the `plotter.axes_ranges` `@property`? I'm probably driving you all nuts with this minor feature implementation lol